cmd/gh/main.go

```diff
@@ -254,6 +254,10 @@ func mainRun() exitCode {
 			ansi.Color(newRelease.URL, "yellow"))
 	}
 
+	if it := os.Getenv("GH_INTEGRATION_TEST"); it != "" {
+		fmt.Fprintf(stderr, "FINISHED")
+	}
```

Comment:
Had to come up with standardized text that we could match on so that the test would know when to stop listening for output.

Reply:
Could we use the fact that `mainRun` finished as a signal that it won't be sending any more output?

Reply:
There might be a way to get this to work, but it is not immediately obvious to me how to do it because of how the `Netflix/go-expect` library works. I can investigate more though.

Reply:
I was able to get something to work here but it relies on `time.Sleep` which seems fragile. Here is a snippet:

```go
	go func() {
		_, _ = c.ExpectString("Would you like to add a remote for the fork?")
		_, _ = c.SendLine("Y")
		_, _ = c.ExpectString("UNKNOWN") // We need to expect something that will never show up so we will keep reading the command output to the buffer
	}()

	// Execute command
	code := mainRun()
	time.Sleep(time.Second) // Assume that 1 second is enough time to for go-expect to flush all output to buffer, will fail if too short of time is given here
	c.Close()
```

Reply:
I decided I don't mind the `time.Sleep` approach here. Going to move forward with that for now.